### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by cjstyles6** [madina0801/JobTrackr#7] [General]
@madina0801 let's work on this project together

**Comment by mdinaya** [comment]
Hi @cjstyles6 ! Thanks for your interest! I'll open many issues tomorrow, feel free to check on them :) If you have some particular ideas or suggestions leave them [here](https://github.com/madina0801/JobTrackr/discussions/categories/ideas) :)

**Reply by cjstyles6** [reply]
Sure

**Comment by MehekFatima** [comment]
Hey @madina0801,

I hope you're doing well. I've submitted a pull request to your repository and would really appreciate it if you could take a look and provide your feedback.

Thank you for your time!
